frontend/institution/institutional_links_mobile.html

```diff
@@ -0,0 +1,34 @@
+<div class="main-links-container">
+        
+    <div class="link-section" ng-if="instLinksCtrl.hasParentInst()">
```

Comment:
these two divs looks pretty much the same, it could be a component and you just have to pass the array with the institutions